.github/workflows/pull-db-tests.yml

```diff
@@ -251,3 +263,16 @@ jobs:
         env:
           TAGS: bindata
           USE_REPO_TEST_DIR: 1
+
+  test-coverage-check:
+    if: needs.files-changed.outputs.backend == 'true' || needs.files-changed.outputs.frontend == 'true'
```

Comment:
I do not think the `frontend` would affect backedn test coverage.

```
            frontend:
              - "**/*.js"
              - "web_src/**"
              - "package.json"
              - "package-lock.json"
```

Reply:
See https://github.com/go-gitea/gitea/pull/22401#discussion_r1227931883 and https://github.com/go-gitea/gitea/pull/25150 for the frontend dependency removal.

Reply:
Yup, that comment seems to have been marked as "resolved", so I started this new conversation.